### DISCUSSION TITLE
Shall I be able to run Draw.io inside Microsoft Teams?

### DISCUSSION THREAD

**Discussion by Petri-X** [jgraph/drawio#2275] [Q&A] [answered]
Hi there,
I just wonder if this instruction is still valid?
[Use draw.io in Microsoft Teams](https://www.diagrams.net/doc/faq/microsoft-teams-diagrams)
I'm asking this, as we have blocked by default all apps, and when trying to search draw.io on the Teams Admin center to be enabled, I cannot find the app at all:
![image](https://user-images.githubusercontent.com/43988151/135233645-4089baf3-3c83-4511-a002-b88a24e9fdb6.png)

Or is there a separate app to be uploaded into Teams Apps?

**Comment by davidjgraph** [comment, marked as answer]
No, it's just the docs have to be released before the app is. The app isn't approved yet. We don't have an ETA.

**Comment by Petri-X** [comment]
Thank you David, and that was quick reply!
